frontend/src/pages/dashboard/components/sidebar/sidebar.tsx

```diff
@@ -31,15 +31,21 @@ const Sidebar: React.FC<Properties> = ({
     >
       <ul className={styles.list}>
         {TABS.map((tab) => (
-          <li key={tab.name}>
+          <li
+            className={
+              selectedTab === tab.name
+                ? getValidClassNames(styles.item, styles.active)
+                : getValidClassNames(styles.item)
+            }
+            key={tab.name}
+          >
             <Button
               label={isCollapsed ? '' : tab.name}
```

Comment:
make label optional in props and add default value = '' in Button component

Reply:
I can't write isCollapsed && tab.name because of type errors

Reply:
You can use this:
`{...(isCollapsed && { label: tab.name })}`